src/components/Masthead.vue

```diff
@@ -3,7 +3,8 @@
     <div class="wrapper">
 
         <header>
-            <div class="logo" v-bind:class="{ login : $route.path==='/login' || $route.path==='/user' || $route.path==='/request' || $route.path==='/change-password' }">
+        
+            <div class="logo" v-bind:class="{ login : $route.path==='/login' || $route.path==='/user' || $route.path==='/request' || $route.path==='/change-password' || $route.path==='/privacy-policy' }">
```

Comment:
@katjam Most likely!

@ob6160 I'm not sure how to do that (hence above code), so if you would like to push up some code for that, please do! :+1: